.env.test

```diff
@@ -45,10 +45,8 @@ ENABLE_FIXED_OTP_ON_REQUEST_FOR_QA=false
 ENABLE_GENERATE_ENCOUNTER_ID_ENDPOINT=true
 ENABLE_PATIENT_FOLLOWUPS=true
 ENABLE_PURGE_ENDPOINT_FOR_QA=false
-ENABLE_SMS_REMINDERS=true
```

Comment:
This must be leftover from previous reminder code -- I don't see any references to this flag anywhere.